src/core/ext/filters/client_channel/resolver/xds/xds_resolver.cc

```diff
@@ -788,9 +789,11 @@ void XdsResolver::StartLocked() {
             grpc_error_std_string(error).c_str());
     std::string error_message;
     grpc_error_get_str(error, GRPC_ERROR_STR_DESCRIPTION, &error_message);
-    Result result;
-    result.service_config = absl::UnavailableError(
+    absl::Status status = absl::UnavailableError(
         absl::StrCat("Failed to create XdsClient: ", error_message));
+    Result result;
+    result.addresses = status;
+    result.service_config = status;
```

Comment:
can use std::move here?

Reply:
There's no need for `std::move()` on an `absl::Status`, because the implementation is basically just a ref-counted pointer.  I believe that clang-tidy complains when you try to use `std::move()`.

Reply:
If it's refcounted, wouldn't std::move help to avoid the unnecessary atomic increment and decrement? I didn't find any information on avoiding std::move() for absl:::Status

Reply:
Looks like this worked, so either I was remembering wrong about clang-tidy not being happy with this, or something has changed since the last time I saw that.

In any case, seems to be fine now.  PTAL.